ipams/builtin/builtin_unix.go

```diff
@@ -11,6 +11,11 @@ import (
 	"github.com/docker/libnetwork/ipamutils"
 )
 
+var (
+	// DefaultAddressPool Store DefaultAddressPool locally to use it during IPAM utils init
+	DefaultAddressPool []*ipamutils.PredefinedPools
```

Comment:
Do we need this to be a global variable ? I dont see a get for this ?

Reply:
made it local variable.